### PR TITLE
Add dashboard resource

### PR DESCRIPTION
**Pull request by zippolyte**
SSIA

### CODE REVIEW

File: datadog-dashboards-dashboard-handler/src/datadog_dashboards_dashboard/handlers.py [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+            model.DatadogCredentials.ApplicationKey,
+            model.DatadogCredentials.ApiURL or "https://api.datadoghq.com",
+            TELEMETRY_TYPE_NAME,
+            __version__,
+    ) as api_client:
+        api_instance = DashboardsApi(api_client)
+        try:
+            dash = api_instance.get_dashboard(dashboard_id)
+            json_dict = ApiClient.sanitize_for_serialization(dash)
+            model.DashboardDefinition = json.dumps(json_dict)
```
Comment (nmuesch) [inline datadog-dashboards-dashboard-handler/src/datadog_dashboards_dashboard/handlers.py:149]: Is it worth catching this too?

**Review by nmuesch — CHANGES_REQUESTED** [review]
Overall looks good. Left some inline comments

File: datadog-dashboards-dashboard-handler/datadog-dashboards-dashboard.json [hunk 2]
```diff
@@ -0,0 +1,79 @@
+{
+  "typeName": "Datadog::Dashboards::Dashboard",
+  "description": "Datadog Dashboard",
```
Comment (nmuesch) [inline datadog-dashboards-dashboard-handler/datadog-dashboards-dashboard.json:3]: I'm also trying to standardize adding the version number of the resource in this field (see PR #97) If that gets approved, can we add the version here

File: datadog-dashboards-dashboard-handler/src/datadog_dashboards_dashboard/handlers.py (hunk 1 above)
Comment (zippolyte) [reply datadog-dashboards-dashboard-handler/src/datadog_dashboards_dashboard/handlers.py:149]: nope, dumping should not raise

**Review by nmuesch — APPROVED** [review]

**Pull request merged**